contents/integration/apache-airflow/index.mdx

```diff
@@ -33,35 +33,43 @@ The integration is implemented using the [openlineage-airflow](https://pypi.org/
 
 ### Prerequisites
 
-To use the OpenLineage Airflow integration, you'll need a running Airflow instance. You'll also need an OpenLineage compatible [HTTP backend](https://github.com/OpenLineage/OpenLineage#scope). In this guide, we'll be using the `openlineage-airflow` library to automatically send OpenLineage events to [Marquez](https://marquezproject.ai).
-
-### Installation
-
-To download and install the latest `openlineage-airflow` library, update the `requirements.txt` file of your running Airflow instance with: 
-
-```
-openlineage-airflow
-```
+To use the OpenLineage Airflow integration, you'll need a running Airflow instance. You'll also need an OpenLineage compatible [HTTP backend](https://github.com/OpenLineage/OpenLineage#scope). In this guide, we'll be using the `openlineage-airflow` library to automatically send OpenLineage events to [Marquez](https://marquezproject.ai). 
 
 ### Configuration
 
-Next, we'll need to specify where we want Airflow to send OpenLineage events. `openlineage-airflow` will use the `OPENLINEAGE_URL` environment variable to send OpenLineage events to Marquez.
+Next, we'll need to specify where we want Airflow to send OpenLineage events. `openlineage-airflow` will use the `OPENLINEAGE_URL` environment variable to send OpenLineage events to Marquez. Optionally, we can also specify a namespace where the lineage events will be stored using the `OPENLINEAGE_NAMESPACE` environment variable.
 
-For example, to send OpenLineage events to a local instance of Marquez, use:
+For example, to send OpenLineage events to a local instance of Marquez with the `dev` namespace, use:
 
 ```bash
 OPENLINEAGE_URL=http://localhost:5000
+OPENLINEAGE_NAMESPACE="dev"
 ```
 
 ### Usage
+#### Airflow 1.10+
+
+First, download and install the latest `openlineage-airflow` library, by updating the `requirements.txt` file of your running Airflow instance with: 
 
-Finally, to begin collecting Airflow DAG metadata with Marquez, make the following change to your DAG:
+```
+openlineage-airflow
+```
+
+Then, to begin collecting Airflow DAG metadata with Marquez, make the following change to your DAG:
 
 ```diff
 - from airflow import DAG
 + from openlineage.airflow import DAG
 ```
 
+#### Airflow 2.0+
```

Comment:
I'd additionally mention the 2.3+ interface: https://github.com/OpenLineage/OpenLineage/pull/508/files#diff-afff38bfef4e3fadfd18de3855a240dc6a7f4d238aa38e3a0da2c29506738a33R46